zstacklib/zstacklib/storage/drbd/resource.py

```
@@ -0,0 +1,429 @@
+    @bash.in_bash
+    @linux.retry(times=90, sleep_time=3)
+    def clear_bits(self):
+        # type: () -> None
+        """Clear the dirty bitmap for initial sync."""
+        bash.bash_errorout("drbdadm new-current-uuid --clear-bitmap %s" % self.name)
+    
+    @bash.in_bash
+    def minor_allocated(self):
+        # type: () -> bool
+        """Check if device minor number is allocated."""
+        r, o, e = bash.bash_roe("drbdadm role %s" % self.name)
```

Ruff (0.15.2)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)